tests/test_astro_yaml.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import os
+import yaml
+
+def test_astro_yaml():
```

Comment:
- The file name `test_astro_yaml.py` is ambiguous. You should probably add a test case to [test_astronomer_registry.py](https://github.com/astronomer/astronomer/blob/966fa36935/tests/test_astronomer_registry.py)
- The function name `test_astro_yaml` is ambiguous.
- This test should be parametrized to support all k8s versions.
- This test doesn't use the existing [`render_chart()`](https://github.com/astronomer/astronomer/blob/966fa36935/tests/helm_template_generator.py#L67) function that is used in all our helm tests.
- This test only checks for valid yaml, not a valid k8s manifest.
- This test doesn't check for presence or validity of the configured value.

Here is a tutorial that explains how these tests should be written: https://github.com/astronomer/astronomer/blob/master/tests/README.md#starting-a-new-pytest-file

See this example for guidance on refactoring this test to include the `render_chart()` function with `values` and `show_only` specified, and assertions made against the values that were given: https://github.com/astronomer/astronomer/blob/966fa36935/tests/test_registry_statefulset.py#L111-L120

One thing I find helpful when writing tests is to put a `breakpoint()` into the new test below `render_chart()` and run `venv/bin/python -m pytest tests --pdb -k registry` to get an interactive shell where I can craft assertions for the test against the `docs`.

I appreciate you taking the effort to add this feature and tests for it. We can't merge this though until test cases that cover the new feature.